fawltydeps/packages.py

```diff
@@ -310,20 +310,45 @@ class TemporaryPipInstallResolver(BasePackageResolver):
 
     @staticmethod
     @contextmanager
-    def temp_installed_requirements(requirements: List[str]) -> Iterator[Path]:
+    def temp_installed_requirements(
+        requirements: List[str],
+        ignore_errors: bool = False,
+        retry_each: bool = False,
+    ) -> Iterator[Path]:
         """Create a temporary venv and install the given requirements into it.
 
         Provide a path to the temporary venv into the caller's context in which
         the given requirements have been `pip install`ed. Automatically remove
         the venv at the end of the context.
+
+        By default, if `pip install` fails, we will propagate the resulting
+        CalledProcessError from here. If 'ignore_errors' is enabled we instead
+        log a warning and continue into the caller's context. It is then up to
+        the caller to handle the missing requirements in the temporary venv.
```

Comment:
If we remove `ignore_errors` option, this desription should explain that `pip install` will stop on a first failed dependency instalation and not instal anything further.

Reply:
Ack. Will update the docstring when I hardcode the new behavior.